README.md

```diff
@@ -62,18 +58,49 @@ optional arguments:
                         is newest to oldest
   -s, --sold            If included, query will return results for sold items
                         as well as active items
+  --email EMAIL         email address to send emails from.If --receiver is not
+                        specified, this email will also be used as the
+                        receiver.
+  --receiver RECEIVER   email address to send the email to. Defaults to
+                        --email value.
+  --exception-receiver EXCEPTION_RECEIVER
+                        email address to send exception emails to. Defaults to
+                        --email value.
+  --smtpserver SMTPSERVER
+                        email SMTP server:port, should be unneeded for gmail,
+                        outlook, hotmail, msn, yahoo, or comcast
+  -t [TIME], --time [TIME]
+                        Number of minutes to wait between searches
+  --head HEAD           Number of lines to include from the listing's
+                        description. If not specified, the entire description
+                        will be included.
+  --char-limit CHAR_LIMIT
+                        Number of characters allowed in the message body.
+                        Listings that exceed the character count will be sent
+                        in additional messages.
+  --exclude-links       Exclude links from message.
+  --load LOAD           Load seen listings from a JSON file. Format is a
+                        dictionary of query search terms to listing links.
+  --save SAVE           Save seen listings to a JSON file. File extension must
```

Comment:
These all could benefit from short options as well. I'd recommend:
* `-h` `--head`
* `-H` `--char-limit`
* `-x` `--exclude-links`
* `-L` `--load`
* `-S` `--save`

Reply:
Done. Had to change some of the short options due to conflicting option names.